webapp/actions/websocket_actions.jsx

```diff
@@ -241,6 +245,12 @@ function handleUserRemovedEvent(msg) {
     }
 }
 
+function handleUserUpdatedEvent(msg) {
+    if (UserStore.getCurrentId() !== msg.user_id) {
+        AsyncClient.getProfiles();
```

Comment:
Yeah, that sounds like a nice compromise. That also avoids the extra request that all clients would be making anyway